tests/e2e/global-prefix.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { INestApplication } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import * as request from 'supertest';
+import { GlobalPrefixModule } from '../graphql/global-prefix.module';
+
+describe('GraphQL (global prefix)', () => {
+  let app: INestApplication;
+
+  beforeEach(async () => {
+    const module = await Test.createTestingModule({
+      imports: [GlobalPrefixModule],
+    }).compile();
+
+    app = module.createNestApplication();
+    app.setGlobalPrefix('/api/v1');
```

Comment:
This option will fail without the starting slash, i.e. `app.setGlobalPrefix('api/v1');`

Reply:
that is indeed a problem! can this be fixed? the official docs ( https://docs.nestjs.com/faq/global-prefix ) indicates the `globalPrefix` **without** the `/` at the start / end..

Reply:
@johannesschobel

See #364

Reply:
haha.. yeah, i was just about to make a PR but then i found yours..